libafl/Cargo.toml

```diff
@@ -236,9 +236,10 @@ ahash = { workspace = true } # The hash function already used in hashbrown
 meminterval = { workspace = true, features = ["serde"] }
 backtrace = { workspace = true, optional = true } # Used to get the stacktrace in StacktraceObserver
 typed-builder = { workspace = true, optional = true } # Implement the builder pattern at compiletime
+fs4 = "0.13.1"
```

Comment:
there should be another dependency on fs2 inside this file
can you delete that?